From 69c14ab6a1a23bb0431bac06632e72add7609505 Mon Sep 17 00:00:00 2001
From: David Joy <djoy@edx.org>
Date: Wed, 8 Jan 2020 15:43:26 -0500
Subject: [PATCH 0016/1021] Fixing breadcrumb key warnings.

---
 src/learning-sequence/Breadcrumbs.jsx | 12 +++++-------
 1 file changed, 5 insertions(+), 7 deletions(-)

diff --git a/src/learning-sequence/Breadcrumbs.jsx b/src/learning-sequence/Breadcrumbs.jsx
index 1828bab..6fda826 100644
--- a/src/learning-sequence/Breadcrumbs.jsx
+++ b/src/learning-sequence/Breadcrumbs.jsx
@@ -1,7 +1,5 @@
-/* eslint-disable react/no-array-index-key */
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
 
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
@@ -15,18 +13,18 @@ const Breadcrumbs = ({
     <nav aria-label="breadcrumb">
       <ol className="list-inline">
         {links.map(({ url, label }, i) => (
-          <>
-            <li key={url} className="list-inline-item">
+          <React.Fragment key={`${label}-${url}`}>
+            <li className="list-inline-item">
               <a href={url} {...(clickHandler && { onClick: clickHandler })}>{label}</a>
             </li>
             {(activeLabel || ((i + 1) < linkCount)) &&
               <li className="list-inline-item" role="presentation" aria-label="spacer">
-                {spacer || <FontAwesomeIcon key={`spacer-${i}`} icon={faChevronRight} />}
+                {spacer || <FontAwesomeIcon icon={faChevronRight} />}
               </li>
             }
-          </>
+          </React.Fragment>
         ))}
-        {activeLabel && <li className="list-inline-item" key="active">{activeLabel}</li>}
+        {activeLabel && <li className="list-inline-item">{activeLabel}</li>}
       </ol>
     </nav>
   );
-- 
2.34.1

