From 07776c217300415ea6d5aecd84596e535fe8479f Mon Sep 17 00:00:00 2001
From: Adam Butterworth <abutterworth@edx.org>
Date: Thu, 9 Jan 2020 18:04:18 -0500
Subject: [PATCH 0018/1021] fix: remove missing import

---
 src/learning-sequence/LearningSequencePage.jsx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/learning-sequence/LearningSequencePage.jsx b/src/learning-sequence/LearningSequencePage.jsx
index e99b03e..408430f 100644
--- a/src/learning-sequence/LearningSequencePage.jsx
+++ b/src/learning-sequence/LearningSequencePage.jsx
@@ -6,7 +6,7 @@ import PageLoading from './PageLoading';
 import messages from './messages';
 
 import CourseBreadcrumbs from './CourseBreadcrumbs';
-import SubSection from './SubSection';
+// import SubSection from './SubSection';
 
 import { useCourseStructure } from './hooks';
 import CourseStructureContext from './CourseStructureContext';
-- 
2.34.1

