From 869eb9da385684e4428279a56c555ab4c10006a9 Mon Sep 17 00:00:00 2001
From: David Joy <djoy@edx.org>
Date: Thu, 16 Jan 2020 10:35:30 -0500
Subject: [PATCH 0042/1021] Fixing disabled-when-active unit button.

---
 src/courseware/sequence/UnitButton.jsx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/courseware/sequence/UnitButton.jsx b/src/courseware/sequence/UnitButton.jsx
index 29cf2f2..3736287 100644
--- a/src/courseware/sequence/UnitButton.jsx
+++ b/src/courseware/sequence/UnitButton.jsx
@@ -19,7 +19,7 @@ export default function UnitButton({
         'btn-outline-primary': !isActive,
         'btn-outline-secondary': isActive,
       })}
-      disabled={isActive}
+
       onClick={onClick}
       title={pageTitle}
     >
-- 
2.34.1

