From 3afac3bcdce7723a525159e53460515b78e5fb84 Mon Sep 17 00:00:00 2001
From: Adam Butterworth <abutterworth@edx.org>
Date: Thu, 23 Jan 2020 17:23:40 -0500
Subject: [PATCH 0049/1021] Allow fullscreen on unit iframe

---
 src/courseware/sequence/Unit.jsx | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/courseware/sequence/Unit.jsx b/src/courseware/sequence/Unit.jsx
index 53b3411..3967633 100644
--- a/src/courseware/sequence/Unit.jsx
+++ b/src/courseware/sequence/Unit.jsx
@@ -12,6 +12,7 @@ export default function Unit({ id, pageTitle }) {
       title={pageTitle}
       ref={iframeRef}
       src={iframeUrl}
+      allowFullScreen
     />
   );
 }
-- 
2.34.1

