From f3e559ad9dcc59e936afaca43ca62b42b15287b5 Mon Sep 17 00:00:00 2001
From: David Joy <djoy@edx.org>
Date: Thu, 5 Mar 2020 10:40:09 -0500
Subject: [PATCH 0064/1021] Adding a TODO about removing the course links
 route.

---
 src/index.jsx | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/index.jsx b/src/index.jsx
index 41ce2dd..6cf899e 100755
--- a/src/index.jsx
+++ b/src/index.jsx
@@ -22,6 +22,7 @@ import CourseContainer from './courseware/CourseContainer';
 import store from './store';
 
 function courseLinks() {
+  // TODO: We should remove these links before we go live for learners.
   return (
     <main className="m-3">
       <ul>
-- 
2.34.1

