From a1eee2d6627daaaf15d386fd5dd7d0b4ae1986ef Mon Sep 17 00:00:00 2001
From: Adam Butterworth <abutterworth@edx.org>
Date: Mon, 4 May 2020 16:14:20 -0400
Subject: [PATCH 0112/1021] Fix IE11 layout issue by setting header flex-basis
 to auto (#58)

---
 src/index.scss | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/index.scss b/src/index.scss
index 9566c0b..36f7467 100755
--- a/src/index.scss
+++ b/src/index.scss
@@ -32,7 +32,7 @@ $primary: #1176B2;
     flex-grow: 1;
   }
   header {
-    flex: 0;
+    flex: 0 0 auto;
 
     .logo {
       display: block;
-- 
2.34.1

