From 65ab77bed34a5133a1613a07a52777d3f80f8407 Mon Sep 17 00:00:00 2001
From: stvn <stvn@mit.edu>
Date: Fri, 15 May 2020 11:58:37 -0700
Subject: [PATCH 0121/1021] Cleanup PORT config

---
 .env.development | 2 +-
 .env.test        | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/.env.development b/.env.development
index d5dca1e..1e130d6 100644
--- a/.env.development
+++ b/.env.development
@@ -1,5 +1,4 @@
 NODE_ENV='development'
-PORT=2000
 ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload'
 BASE_URL='localhost:2000'
 CREDENTIALS_BASE_URL='http://localhost:18150'
@@ -11,6 +10,7 @@ LOGIN_URL='http://localhost:18000/login'
 LOGOUT_URL='http://localhost:18000/logout'
 MARKETING_SITE_BASE_URL='http://localhost:18000'
 ORDER_HISTORY_URL='localhost:1996/orders'
+PORT=2000
 REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
 SEGMENT_KEY=null
 SITE_NAME='edX'
diff --git a/.env.test b/.env.test
index 4b8d98e..9f2a03b 100644
--- a/.env.test
+++ b/.env.test
@@ -9,6 +9,7 @@ LOGIN_URL='http://localhost:18000/login'
 LOGOUT_URL='http://localhost:18000/logout'
 MARKETING_SITE_BASE_URL='http://localhost:18000'
 ORDER_HISTORY_URL='localhost:1996/orders'
+PORT=2000
 REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
 SEGMENT_KEY=null
 SITE_NAME='edX'
-- 
2.34.1

