From 93ccdf829b6be7c7c8316c912403d5b14db2bc92 Mon Sep 17 00:00:00 2001
From: stvn <stvn@mit.edu>
Date: Fri, 15 May 2020 11:59:07 -0700
Subject: [PATCH 0122/1021] Fix mismatched test BASE_URL

---
 .env.test | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.env.test b/.env.test
index 9f2a03b..effe130 100644
--- a/.env.test
+++ b/.env.test
@@ -1,5 +1,5 @@
 ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload'
-BASE_URL='localhost:1995'
+BASE_URL='localhost:2000'
 CREDENTIALS_BASE_URL='http://localhost:18150'
 CSRF_TOKEN_API_PATH='/csrf/api/v1/token'
 ECOMMERCE_BASE_URL='http://localhost:18130'
-- 
2.34.1

