From b2fa93af139136ed6a4abe481ff4932873eb17d4 Mon Sep 17 00:00:00 2001
From: stvn <stvn@mit.edu>
Date: Fri, 15 May 2020 11:59:22 -0700
Subject: [PATCH 0123/1021] Set NODE_ENV in the test environment

---
 .env.test | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.env.test b/.env.test
index effe130..f6b129a 100644
--- a/.env.test
+++ b/.env.test
@@ -1,3 +1,4 @@
+NODE_ENV='test'
 ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload'
 BASE_URL='localhost:2000'
 CREDENTIALS_BASE_URL='http://localhost:18150'
-- 
2.34.1

