From e101b41c0840f13b6c143f70d96aa78bc879d966 Mon Sep 17 00:00:00 2001
From: Michael Terry <mterry@edx.org>
Date: Thu, 21 May 2020 12:03:09 -0400
Subject: [PATCH 0130/1021] Whoops, add one-liner missing from github in last
 commit (#72)

---
 src/courseware/CoursewareContainer.jsx | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/courseware/CoursewareContainer.jsx b/src/courseware/CoursewareContainer.jsx
index c418241..488edaa 100644
--- a/src/courseware/CoursewareContainer.jsx
+++ b/src/courseware/CoursewareContainer.jsx
@@ -204,6 +204,7 @@ export default function CoursewareContainer() {
     <TabPage
       activeTabSlug="courseware"
       courseId={courseId}
+      unitId={routeUnitId}
     >
       <Course
         courseId={courseId}
-- 
2.34.1

