From 0d290827933adce85ea7f8d78c1166dcd4f3b6a5 Mon Sep 17 00:00:00 2001
From: Zainab Amir <zainab.amir@arbisoft.com>
Date: Thu, 3 Dec 2020 15:39:16 +0500
Subject: [PATCH 0312/1021] Add suppression to username in header (#287)

---
 src/course-header/Header.jsx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/course-header/Header.jsx b/src/course-header/Header.jsx
index b91375e..bd31ecf 100644
--- a/src/course-header/Header.jsx
+++ b/src/course-header/Header.jsx
@@ -88,7 +88,7 @@ function Header({
         <Dropdown className="user-dropdown">
           <Dropdown.Toggle variant="light">
             <FontAwesomeIcon icon={faUserCircle} className="d-md-none" size="lg" />
-            <span className="d-none d-md-inline">
+            <span data-hj-suppress className="d-none d-md-inline">
               {authenticatedUser.username}
             </span>
           </Dropdown.Toggle>
-- 
2.34.1

