From 7fed8db02a79514a6274670fb93d36f1bbb4b037 Mon Sep 17 00:00:00 2001
From: Matthew Piatetsky <mpiatetsky@edx.org>
Date: Mon, 21 Dec 2020 13:01:47 -0500
Subject: [PATCH 0353/1021] fix button interactivity (#328)

---
 src/courseware/course/sequence/Sequence.jsx | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/courseware/course/sequence/Sequence.jsx b/src/courseware/course/sequence/Sequence.jsx
index 606c509..c93b136 100644
--- a/src/courseware/course/sequence/Sequence.jsx
+++ b/src/courseware/course/sequence/Sequence.jsx
@@ -85,6 +85,7 @@ function REV1512Flyover({ toggleREV1512Flyover }) {
           }}
         />
       </div>
+      <div className="rev-1512-optimizely-flyover-container" />
     </div>
   );
 }
@@ -172,6 +173,7 @@ function REV1512FlyoverMobile({ toggleREV1512Flyover }) {
           }}
         />
       </div>
+      <div className="rev-1512-optimizely-flyover-container" />
     </div>
   );
 }
-- 
2.34.1

