From aa56239f545633524f5f322b899500500d0abc97 Mon Sep 17 00:00:00 2001
From: Matthew Piatetsky <mpiatetsky@edx.org>
Date: Wed, 6 Jan 2021 11:37:34 -0500
Subject: [PATCH 0361/1021] add empty div to fix bug with moving flyover toggle

---
 .../course/sequence/sequence-navigation/SequenceNavigation.jsx   | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/courseware/course/sequence/sequence-navigation/SequenceNavigation.jsx b/src/courseware/course/sequence/sequence-navigation/SequenceNavigation.jsx
index 6c5aaf5..5825e5d 100644
--- a/src/courseware/course/sequence/sequence-navigation/SequenceNavigation.jsx
+++ b/src/courseware/course/sequence/sequence-navigation/SequenceNavigation.jsx
@@ -78,6 +78,7 @@ function SequenceNavigation({
       </Button>
       {renderUnitButtons()}
       {renderNextButton()}
+      <div className="rev1512ToggleFlyoverSequenceLocation" />
     </nav>
   );
 }
-- 
2.34.1

