From acd2cc3222b0c6662086446beac022aa582b131e Mon Sep 17 00:00:00 2001
From: Carla Duarte <cduarte@edx.org>
Date: Wed, 27 Jan 2021 13:59:06 -0500
Subject: [PATCH 0375/1021] AA-638: course completion bug (#352)

---
 src/tab-page/TabContainer.jsx | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/tab-page/TabContainer.jsx b/src/tab-page/TabContainer.jsx
index ed634b3..7e06569 100644
--- a/src/tab-page/TabContainer.jsx
+++ b/src/tab-page/TabContainer.jsx
@@ -32,6 +32,7 @@ export default function TabContainer(props) {
       activeTabSlug={tab}
       courseId={courseId}
       courseStatus={courseStatus}
+      metadataModel={`${slice}Meta`}
     >
       {children}
     </TabPage>
-- 
2.34.1

