From 774b7bb1fc4c2e9a77c3e492c6818afe3d29a085 Mon Sep 17 00:00:00 2001
From: Matthew Carter <mcarter@edx.org>
Date: Fri, 12 Mar 2021 14:44:38 -0500
Subject: [PATCH 0404/1021] Set flyover visability default to true (#388)

---
 src/experiments/mm-p2p/index.jsx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/experiments/mm-p2p/index.jsx b/src/experiments/mm-p2p/index.jsx
index 70437cf..89d4cfa 100644
--- a/src/experiments/mm-p2p/index.jsx
+++ b/src/experiments/mm-p2p/index.jsx
@@ -150,7 +150,7 @@ const initCoursewareMMP2P = (courseId, sequenceId, unitId) => {
   };
 
   const [isMMP2PFlyoverVisible, setMMP2PFlyoverVisible] = useState(
-    isMobile() ? false : defaultWindowVal(MMP2PKeys.flyoverVisible, false),
+    isMobile() ? false : defaultWindowVal(MMP2PKeys.flyoverVisible, true),
   );
   const flyover = {
     isVisible: isMMP2PFlyoverVisible,
-- 
2.34.1

