From 43033ddc91593825bf40b81f38b8786c6d52c3cc Mon Sep 17 00:00:00 2001
From: Jansen Kantor <jkantor@edx.org>
Date: Wed, 30 Jun 2021 09:07:59 -0400
Subject: [PATCH 0516/1021] move mmp2p useModel call and use outline model
 (#509)

---
 src/experiments/mm-p2p/index.jsx | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/experiments/mm-p2p/index.jsx b/src/experiments/mm-p2p/index.jsx
index 33dd70b..2702e4e 100644
--- a/src/experiments/mm-p2p/index.jsx
+++ b/src/experiments/mm-p2p/index.jsx
@@ -108,9 +108,9 @@ const initHomeMMP2P = (courseId) => {
   const setMMP2POptions = createWindowStateSetter(_setMMP2POptions, MMP2PKeys.state);
   const setMMP2PAccess = createWindowStateSetter(_setMMP2PAccess, MMP2PKeys.access);
 
-  const loadAccess = () => {
-    const { accessExpiration, verifiedMode } = useModel('coursewareMeta', courseId);
+  const { accessExpiration, verifiedMode } = useModel('outline', courseId);
 
+  const loadAccess = () => {
     if (accessExpiration !== null && accessExpiration !== undefined) {
       setMMP2PAccess({
         isAudit: true,
-- 
2.34.1

