From febc0cae0b4fcdefef250066fc84c35316c7170a Mon Sep 17 00:00:00 2001
From: Carla Duarte <cduarte@edx.org>
Date: Thu, 16 Dec 2021 13:54:32 -0500
Subject: [PATCH 0760/1021] fix: course breadcrumb styling (#776)

---
 src/courseware/course/CourseBreadcrumbs.jsx | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/courseware/course/CourseBreadcrumbs.jsx b/src/courseware/course/CourseBreadcrumbs.jsx
index 8e8a6c8..b7b41e7 100644
--- a/src/courseware/course/CourseBreadcrumbs.jsx
+++ b/src/courseware/course/CourseBreadcrumbs.jsx
@@ -19,7 +19,7 @@ function CourseBreadcrumb({
   return (
     <>
       {withSeparator && (
-        <li className="mx-2 text-primary-500 text-truncate text-nowrap" role="presentation" aria-hidden>/</li>
+        <li className="col-auto p-0 mx-2 text-primary-500 text-truncate text-nowrap" role="presentation" aria-hidden>/</li>
       )}
 
       <li style={{
@@ -124,7 +124,7 @@ export default function CourseBreadcrumbs({
   return (
     <nav aria-label="breadcrumb" className="my-4 d-inline-block col-sm-10">
       <ol className="list-unstyled d-flex  flex-nowrap align-items-center m-0">
-        <li className="list-unstyled d-flex m-0">
+        <li className="list-unstyled col-auto m-0 p-0">
           <a
             href={`/course/${courseId}/home`}
             className="flex-shrink-0 text-primary"
-- 
2.34.1

