From 550b15a16c0c52eb20346a061e0276c7a8381af6 Mon Sep 17 00:00:00 2001
From: edX requirements bot <testeng+edx-requirements-bot@edx.org>
Date: Thu, 17 Mar 2022 08:45:06 -0400
Subject: [PATCH 0881/1021] fix: transifex migration to new client

---
 .tx/config | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/.tx/config b/.tx/config
index 84d8104..1f2663d 100644
--- a/.tx/config
+++ b/.tx/config
@@ -1,8 +1,9 @@
 [main]
 host = https://www.transifex.com
 
-[edx-platform.frontend-app-learning]
+[o:open-edx:p:edx-platform:r:frontend-app-learning]
 file_filter = src/i18n/messages/<lang>.json
 source_file = src/i18n/transifex_input.json
 source_lang = en
-type = KEYVALUEJSON
+type        = KEYVALUEJSON
+
-- 
2.34.1

