From b32817b3dd71eaa65e04955574af6be2c7609df1 Mon Sep 17 00:00:00 2001
From: Usama Sadiq <usama.sadiq@arbisoft.com>
Date: Mon, 4 Apr 2022 16:04:32 +0500
Subject: [PATCH 0887/1021] build: update transifex pull translations command
 (#903)

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5ef6e06..955408c 100644
--- a/Makefile
+++ b/Makefile
@@ -44,7 +44,7 @@ push_translations:
 
 # Pulls translations from Transifex.
 pull_translations:
-	tx pull -f --mode reviewed --language=$(transifex_langs)
+	tx pull -f --mode reviewed --languages=$(transifex_langs)
 
 # This target is used by Travis.
 validate-no-uncommitted-package-lock-changes:
-- 
2.34.1

