From f3c80ed39b79c6e00b63b185aaaa28e2692d0f54 Mon Sep 17 00:00:00 2001
From: Leangseu Kim <lkim@edx.org>
Date: Tue, 20 Dec 2022 13:39:18 -0500
Subject: [PATCH 0953/1021] fix: text in goal modal

---
 .../celebration/WeeklyGoalCelebrationModal.jsx | 18 ++++++++++--------
 1 file changed, 10 insertions(+), 8 deletions(-)

diff --git a/src/courseware/course/celebration/WeeklyGoalCelebrationModal.jsx b/src/courseware/course/celebration/WeeklyGoalCelebrationModal.jsx
index 4b278bb..4d04f16 100644
--- a/src/courseware/course/celebration/WeeklyGoalCelebrationModal.jsx
+++ b/src/courseware/course/celebration/WeeklyGoalCelebrationModal.jsx
@@ -57,14 +57,16 @@ function WeeklyGoalCelebrationModal({
             className="mr-2"
             style={{ height: '21px', width: '22px' }}
           />
-          <FormattedMessage
-            id="learning.celebration.setGoal"
-            defaultMessage="Setting a goal can help you {strongText} in your course."
-            description="It explain the advantages of setting goal"
-            values={{
-              strongText: (<strong>achieve higher performance</strong>),
-            }}
-          />
+          <div>
+            <FormattedMessage
+              id="learning.celebration.setGoal"
+              defaultMessage="Setting a goal can help you {strongText} in your course."
+              description="It explain the advantages of setting goal"
+              values={{
+                strongText: (<strong>achieve higher performance</strong>),
+              }}
+            />
+          </div>
         </div>
       </>
     </StandardModal>
-- 
2.34.1

