From 33e3765b1963347e6df881b6d85650ba8b180b6d Mon Sep 17 00:00:00 2001
From: Varsha <46579265+varshamenon4@users.noreply.github.com>
Date: Wed, 22 Feb 2023 11:40:24 -0500
Subject: [PATCH 0991/1021] build: add exams url to envs (#1066)

---
 .env             | 1 +
 .env.development | 1 +
 .env.test        | 1 +
 3 files changed, 3 insertions(+)

diff --git a/.env b/.env
index c882fde..f781fa0 100644
--- a/.env
+++ b/.env
@@ -15,6 +15,7 @@ ECOMMERCE_BASE_URL=''
 ENABLE_JUMPNAV='true'
 ENABLE_NOTICES=''
 ENTERPRISE_LEARNER_PORTAL_HOSTNAME=''
+EXAMS_BASE_URL=''
 FAVICON_URL=''
 IGNORED_ERROR_REGEX=''
 INSIGHTS_BASE_URL=''
diff --git a/.env.development b/.env.development
index f191351..603564b 100644
--- a/.env.development
+++ b/.env.development
@@ -15,6 +15,7 @@ ECOMMERCE_BASE_URL='http://localhost:18130'
 ENABLE_JUMPNAV='true'
 ENABLE_NOTICES=''
 ENTERPRISE_LEARNER_PORTAL_HOSTNAME='localhost:8734'
+EXAMS_BASE_URL='http://localhost:18740'
 FAVICON_URL=https://edx-cdn.org/v3/default/favicon.ico
 IGNORED_ERROR_REGEX=''
 LANGUAGE_PREFERENCE_COOKIE_NAME='openedx-language-preference'
diff --git a/.env.test b/.env.test
index 147c12c..ff7ddff 100644
--- a/.env.test
+++ b/.env.test
@@ -15,6 +15,7 @@ ECOMMERCE_BASE_URL='http://localhost:18130'
 ENABLE_JUMPNAV='true'
 ENABLE_NOTICES=''
 ENTERPRISE_LEARNER_PORTAL_HOSTNAME='localhost:8734'
+EXAMS_BASE_URL='http://localhost:18740'
 FAVICON_URL=https://edx-cdn.org/v3/default/favicon.ico
 IGNORED_ERROR_REGEX=''
 LANGUAGE_PREFERENCE_COOKIE_NAME='openedx-language-preference'
-- 
2.34.1

